### DISCUSSION TITLE
Monitoring improvements

### DISCUSSION THREAD

**Discussion by eduardogsilva** [eduardogsilva/routerfleet#22] [Announcements]
Hello!

I’m excited to announce some important improvements to the RouterFleet monitoring tool! 

- Faster Monitoring for New Hosts: The tool now detects and begins monitoring new hosts much quicker than before.

- Host Availability History: We are now tracking and recording the availability history of all monitored hosts. This will help in diagnosing network issues and improving network reliability.

These updates are designed to make RouterFleet more responsive and informative. I’m looking forward to hearing your thoughts and feedback on these improvements!

Cheers,
Eduardo Silva

**Comment by jakes0ns** [comment]
Thanks Eduardo! 

I will be following this very closely 

Jake
